src/traces.jl

```diff
@@ -32,6 +32,8 @@ Base.pop!(t::Trace) = pop!(t.x)
 Base.popfirst!(t::Trace) = popfirst!(t.x)
 Base.empty!(t::Trace) = empty!(t.x)
 
+fetch(t::Trace, inds) = t[inds]
```

Comment:
It's intentional, because I created it for internal use to avoid changing the behavior of getindex for NormalizedTrace. But it can be exported if necessary I guess.